gqlalchemy/loaders.py

```diff
@@ -0,0 +1,570 @@
+# Copyright (c) 2016-2022 Memgraph Ltd. [https://memgraph.com]
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from string import Template
+
+from . import Memgraph
+from .query_builder import QueryBuilder, Unwind
```

Comment:
I know it's inside gqlalchemy. A preference though only.

Reply:
We should then agree on this and correct it everywhere. Besides that, we need to add that in contribution guide if we decide.

Reply:
We will resolve this in subsequent PR